packages/peregrine/lib/talons/WishlistPage/useCreateWishlist.js

```diff
@@ -0,0 +1,47 @@
+import { useState, useCallback } from 'react';
+
+/**
+ * @function
```

Comment:
Why @function js doc ?

Reply:
This is part of the new Docs initiate we have started for the code. I modeled this after @jcalcaben's previous PR. @jcalcaben is this as expected?

Reply:
@larsroettig this is used to help me auto-generate reference docs from the source code

@revanth0212 yes, this looks like the proper format